modules/mono/glue/Managed/Files/Color.cs

```diff
@@ -104,6 +105,155 @@ public float v
             }
         }
 
+        private static Dictionary<String, Color> _namedColors = new Dictionary<string, Color>()
```

Comment:
I would add a comment here explaining this comes from https://github.com/godotengine/godot/blob/master/core/color_names.inc to make maintaining easier. Alternatively we could make ColorN use an internal call to avoid having to lists.

Reply:
`String` vs `string` inconsistency here?